phoenix-core/src/it/java/org/apache/phoenix/end2end/ParameterizedIndexUpgradeToolIT.java

```diff
@@ -87,9 +87,9 @@ public class ParameterizedIndexUpgradeToolIT extends BaseTest {
     private static final String [] TABLE_LIST_NAMESPACE_SIMPLIFIED = new String[1];
 
     private static final String [] TRANSACTIONAL_TABLE_LIST = new String[1];
-
+    private static String tmpPath = System.getProperty("java.io.tmpdir");
```

Comment:
This is supposed to be value of tmpdir after minicluster has been started.
(But it is not, see my previous commenft()